# generated by patch-package 6.4.11
#
# command:
#   npx patch-package rollup-plugin-string
#
# declared package:
#   rollup-plugin-string: 3.0.0
#
diff --git a/node_modules/rollup-plugin-string/index.js b/node_modules/rollup-plugin-string/index.js
index e67387f..7495e80 100644
--- a/node_modules/rollup-plugin-string/index.js
+++ b/node_modules/rollup-plugin-string/index.js
@@ -13,7 +13,7 @@ function string(opts = {}) {
     transform(code, id) {
       if (filter(id)) {
         return {
-          code: `export default ${JSON.stringify(code)};`,
+          code: 'export default `\n' + code + '`;',
           map: { mappings: "" }
         };
       }
